src/modules/users/users.service.ts

```diff
@@ -1,28 +1,126 @@
-import { Injectable } from '@nestjs/common';
+import { Wishlist } from '#/modules/wishlists/entities/wishlist.entity';
+import { UpdateProfileDto } from './dto/requests/update-profile.dto';
+import {
+  ForbiddenException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Account } from './entities/account.entity';
+import { Profile } from './entities/profile.entity';
+import { Repository } from 'typeorm';
+import { Roles } from '#/shared/constants/user-roles.constant';
 
 @Injectable()
 export class UserService {
-  constructor(@InjectRepository(Account) private userRepo: Account) {}
+  constructor(
+    @InjectRepository(Account) private userAccountRepo: Repository<Account>,
+    @InjectRepository(Profile) private userProfileRepo: Repository<Profile>,
+    @InjectRepository(Wishlist) private wishlistRepo: Repository<Wishlist>,
```

Comment:
So far we used full names like ```bookingRepository``` and ```wishlistRepository```. Let's be consistent

Reply:
Fixed